src/instance/minecraft/commands/SkillCommand.ts

```diff
@@ -0,0 +1,72 @@
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+import { Client, SKYBLOCK_SKILL_DATA, SkyblockMember } from 'hypixel-api-reborn'
+import { HypixelSkyblock } from '../../../type/HypixelApiType'
+
+export default {
+  triggers: ['skill', 's'],
+  enabled: true,
+  handler: async function (context: ChatCommandContext): Promise<string> {
+    const givenUsername = context.args[1] ?? context.username
+    const skills = [
+      'farming',
+      'mining',
+      'combat',
+      'foraging',
+      'fishing',
+      'enchanting',
+      'alchemy',
+      'carpentry',
+      'runecrafting',
+      'taming',
+      'average'
+    ]
+    const skill = context.args[0]
```

Comment:
Reorder the lines where arguments are checked starting from 0. Basically put it above line 9.